kapitan/inputs/jsonnet.py

```diff
@@ -53,7 +53,7 @@ def _search_imports(cwd, imp):
         output_obj = json.loads(json_output)
 
         output = kwargs.get("output", "yaml")
-        prune = kwargs.get("prune", False)
+        prune = kwargs.get("prune_input", False)
```

Comment:
Not sure if this will work. I have the impression `kwargs` here is referencing `--prune` from the command line.